x-pack/plugins/features/common/kibana_feature.ts

```diff
@@ -98,6 +98,11 @@ export interface KibanaFeatureConfig {
    */
   alerting?: readonly string[];
 
+  /**
+   * If your feature grants access to specific case types, you can specify them here to control visibility based on the current space.
+   */
+  cases?: readonly string[];
```

Comment:
@pgayvallet I agree that this isn't the best design, but it's the only option available to us right now. I think it's absolutely worth discussing this further, but outside the scope of this PR. I view this as a limitation of our authorization model to an extent, and is something we would ideally address as part of https://github.com/elastic/kibana/issues/95513. We need solution-specific constructs within the features plugin because the SO authorization model only works on a per-type basis, and alerting/cases both require more granularity than that.

Reply:
>  I think it's absolutely worth discussing this further, but outside the scope of this PR

Yeah, let's open a separate issue. It shouldn't block the current PR as we don't provide a better alternative at the moment.  Who is the best person to describe the problem space? @legrego maybe?